Core/src/IfcGeometryConverter/ProfileConverter.h

```diff
@@ -1393,6 +1331,18 @@ namespace OpenInfraPlatform {
 				}
 			}
 
+			void SearchExistingVertex(std::stringstream ID, uint32_t index, carve::geom::vector<3> position) const

```

Comment:
Also, please pass `std::string` instead of `std::stringstream` and than use the `ID.str()` when calling the function.

Reply:
Resolved: 1b6fe62